src/main/java/io/jenkins/plugins/tuleap_api/client/internals/TuleapApiClient.java

```diff
@@ -69,14 +69,13 @@ public Boolean checkAccessKeyIsValid(Secret secret) {
                 throw new InvalidTuleapResponseException(response);
             }
 
-            return ImmutableList.copyOf(
-                objectMapper
+            return new ArrayList<>(objectMapper
```

Comment:
Why do you not use Collections.unmodifiableList() here ? (like in AccessChecker.java)